nixos/modules/security/acme/default.nix

```
@@ -721,6 +749,19 @@ in {
         '';
       };
 
+      maxConcurrentRenewals = mkOption {
+        type = types.int;
+        default = 5;
+        description = lib.mdDoc ''
+          The maximum number of certificate renewals to run in parallel.
+          This is achieved by chaining the services together with Systemd unit
```

```suggestion
          This is achieved by chaining the services together with systemd unit
```

see [Spelling](https://www.freedesktop.org/wiki/Software/systemd/) ;)